frame/contracts/src/benchmarking/mod.rs

```diff
@@ -170,7 +178,7 @@ where
 
 /// The funding that each account that either calls or instantiates contracts is funded with.
 fn caller_funding<T: Config>() -> BalanceOf<T> {
-	BalanceOf::<T>::max_value() / 2u32.into()
+	BalanceOf::<T>::max_value() / 10_000u32.into()
```

Comment:
Minting can now overflow, so we can't abuse of the funding and needed to reduce it
https://github.com/paritytech/substrate/blob/d58481569817f4a2074dbab014fa629dea7e0ec9/frame/support/src/traits/tokens/fungible/regular.rs#L242-L247